model/xgboost/examples/bostonregression/bostonregression.py

```diff
@@ -0,0 +1,44 @@
+from dffml.feature.feature import Feature, Features
```

Comment:
We try to import from the top level package in examples unless it's from noasync (because we default to the async ones coming from the main package).

```suggestion
from dffml import Feature, Features
```